app/commands.py

```diff
@@ -545,3 +547,18 @@ def populate_redis_template_usage(service_id, day):
             current_app.config['EXPIRE_CACHE_EIGHT_DAYS'],
             raise_exception=True
         )
+
+
+@notify_command(name='rebuild-ft-billing-for-month-and-service')
+@click.option('-s', '--service_id', required=True, type=click.UUID)
+@click.option('-d', '--day', help="The date to recalculate, as YYYY-MM-DD", required=True,
+              type=click_dt(format='%Y-%m-%d'))
+def rebuild_ft_billing_for_month_and_service(service_id, day):
```

Comment:
Date or day? and might be worth adding a comment with an example?

Reply:
It recalculates a whole day (using BST)